scipy/stats/_morestats.py

```diff
@@ -2645,6 +2646,9 @@ def levene(*samples, center='median', proportiontocut=0.05):
     return LeveneResult(W, pval)
 
 
+@_deprecated("'binom_test' is deprecated in favour of"
+             " 'binomtest' from version 1.7.0 and will"
+             " be removed in Scipy 1.12.0.")
 def binom_test(x, n=None, p=0.5, alternative='two-sided'):
     """Perform a test that the probability of success is p.
 
```

Comment:
Cannot comment bellow, but just seeing now that we should remove the note since we are using a proper `deprecated` marker now. Sorry I missed this because it was folded...

Reply:
Do you mean remove [this](https://github.com/scipy/scipy/pull/16307/files#diff-3b91e6b24fd2dd87d04d589edb6867400ee2a9bbb2813bf0691e991078a600d3R2662)? It seems relevant for the docs, doesn't it?

Reply:
I mean this:
```
Note: `binom_test` is deprecated; it is recommended that `binomtest`
    be used instead.
```
It's a duplicate to what you are adding here.